cdap-explore/src/main/java/co/cask/cdap/explore/executor/ExploreExecutorHttpHandler.java

```diff
@@ -105,26 +101,25 @@ public void enableStream(HttpRequest request, HttpResponder responder,
     try {
       Id.Stream streamId = Id.Stream.from(namespaceId, streamName);
 
-      String streamLocationURI;
+      Location streamLocation;
       StreamConfig streamConfig;
       try {
         streamConfig = streamAdmin.getConfig(streamId);
-        Location streamLocation = streamConfig.getLocation();
+        streamLocation = streamConfig.getLocation();
         if (streamLocation == null) {
           responder.sendString(HttpResponseStatus.NOT_FOUND, "Could not find location of stream " + streamName);
           return;
         }
-        streamLocationURI = streamLocation.toURI().toString();
       } catch (IOException e) {
         LOG.info("Could not find stream {} to enable explore on.", streamName, e);
         responder.sendString(HttpResponseStatus.NOT_FOUND, "Could not find stream " + streamName);
         return;
       }
 
-      LOG.debug("Enabling explore for stream {} at location {}", streamName, streamLocationURI);
+      LOG.debug("Enabling explore for stream {} at location {}", streamName, streamLocation);
```

Comment:
I remember seeing ``Location.toString()`` returning the object reference as output. Can you please check?